src/components/Model3DViewer/stories/full.md

```diff
@@ -81,6 +81,7 @@ function ExampleComponent(props) {
 | `assetId`                  | id of asset to highlight                                                                             | `number`                                                                            |         |
 | `boundingBox`              | bounding box object, that describes dimension of viewed asset nodes                                  | `THREE.Box3`                                                                        |         |
 | `cache`                    | object for caching 3D viewers instances                                                              | `{ [name:string]: any }`                                                            |         |
+| `highlightNodes`           | enable node highlighting behaviour                                                                    | `boolean`                                                                           | true    |
```

Comment:
not very clear what it does. if I never seen it I wouldn't understand, does it highlights all nodes of the asset?

Reply:
It highlights all nodes that are mapped to some asset - @mrtraser please correct if I am wrong?

@polomani - Maybe we should name the property `highlightMappedNodes` to make the intent obvious?

Reply:
`enable node highlighting behaviour` -> `enable default highlighting logic for mapped to provided assetId nodes`

Reply:
Updated the property description to `enable default highlighting of 3D nodes that are associated with some asset`